### DISCUSSION TITLE
[FEATURE REQUEST] ¿Cómo convertir el formulario enviado en un pdf?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#134] [3. Ideas & Feature Requests] [answered]
Hola

Quisiera hacer lo siguiente:

En el momento que mande un formulario, el sistema me envié un mensaje con el pdf del screen de las respuestas del formulario (lo que se obtiene en reports). Se que debo el pb pdf y un pb messages para enviar el pdf adjunto. Creo que tambien requiero un template para hacer el pdf, pero no sé como se hace el template o en dónde se hace?.

Por qué no quiero usar reports?
Fundamento: porque debe haber 2 roles diferentes, 1 quien ve los formularios de la task y un administrador que vea todo (esto corresponde al área de salud)

Gracias :>

**Comment by acedward** [comment, marked as answer]
Si, un template es necesario para generar un PDF. 
Actualmente, no tenemos un buen mecanismo para dar soporte a esto.
Entonces vamos a hace dos cosas
1.  Si es urgente, por favor levanta con tu representante Cotalker @jdurandpaz para que se te brinde soporte interno para el template. Básicamente vas a necesitar entregar una plantilla HTML base, donde algunos valores serán remplazados.
2.  Por otra parte, tenemos que mejorar la experiencia de poder crear templates. Vamos a revisitar un proyecto que tenemos en standby hace un tiempo.

**Reply by NicolasPLG** [reply]
gracias @acedward por las respuestas. Cerraré esta discusión y haré el procedimiento indicado.

**Reply by z0rro2k** [reply]
hola. si solucionaron esto, es posible que compartan como se hizo? gracias

**Reply by NicolasPLG** [reply]
hola @z0rro2k esto se resuelve mediante un faas, como lo indiqué anteriormente.

Te recomiendo levantar otra discusión anexando este.

=>
